config/custom_items.txt

```diff
@@ -140,6 +140,11 @@ item_path: /obj/item/clothing/suit/storage/seromi/cloak/fluff/kita
 }
 # ######## D CKEYS
 {
+ckey: dameonowen
+character_name: Dameon Owen
+item_path: /obj/item/weapon/reagent_containers/food/snacks/cookie/mysterious
+}
```

Comment:
Only thing i recommend is just adding a enter after this because of consistency with the rest of the file